main/lxc/lxc.confd

```diff
@@ -0,0 +1,2 @@
+# enable cgroup for systemd based containers
+#SYSTEMD_CTR=Y
```

Comment:
Please use lowercase for variable name.

What does `_ctr` mean in this context? It’s not very clear IMO.

Reply:
have changed the varialble to:

`systemd_container`

& rebased

Reply:
& checksums fixed